### DISCUSSION TITLE
Check the width of PicketFence

### DISCUSSION THREAD

**Discussion by antoinegobert** [jrkerns/pylinac#438] [Q&A]
Hello,

I would like to know if it would be easy to custom the picket fence code to analyse the position of the the fence and his width with proper tolerance to each ?
I'm not an good programmer, so sorry if the question is obvious :)

Thanks

**Comment by jrkerns** [comment]
I don't quite understand what you're asking. Can you ask it differently? Pylinac is open source so you can fork and change any code that you like. Also, for general questions like these, it's better to ask on the forum so that other people can see an possibly answer: https://groups.google.com/g/pylinac

**Reply by antoinegobert** [reply]
Sorry to post here, I had no idea about google's forum. I will post there and try to better explain my request.
Thanks
